multisig-wallet/project/contracts/multisig-contract/src/data_structures/hashing.sw

```diff
@@ -1,41 +1,137 @@
 library;
 
 use ::data_structures::user::User;
+use std::{bytes::Bytes, constants::ZERO_B256};
 
-pub struct Transaction {
+impl Bytes {
```

Comment:
This functionality is needed to hash the structs containing `Bytes` fields. It has not been added to the standard library as we are looking to move away from relying on memory representation for serialization as per the following from an internal discussion: 

> Relying on the memory representation for serialization is a nice hack, but we're actively trying to not support this to replace it with some defined encoding.

However this defined coding has not yet been implemented and the functionality is required for serialisation of Bytes here

Reply:
I see. This definitely should not be at the application level. We don't want to encourage users to play with `asm` blocks. There should be an issue raised to move this to some external place so that we can import it. Perhaps sway libs.